src/worker/server/server.go

```diff
@@ -215,6 +216,34 @@ func Main() (err error) {
 		return err
 	}
 
+	// cleaning old logs and setting up new logs
```

Comment:
I'm wondering if we really need to support logging to files at all at this point?  Probably something good to do eventually, but we do already have a detach mode ("-d") that launches the worker in the background and redirects stdout to worker.out